sufia-models/lib/generators/sufia/models/templates/config/sufia.rb

```diff
@@ -85,6 +85,9 @@
   # Requires a Google Analytics id and OAuth2 keyfile.  See README for more info
   config.analytics = false
 
+  # Specify file for logging upload file errors.
+  config.upload_logger = 'log/upload_error.log'
```

Comment:
I'd like to see this feature off by default, for folks who already use Sufia and expect all their logging to go into one place.